cfgov/jinja2/v1/_includes/molecules/text_introduction.html

```diff
@@ -0,0 +1,40 @@
+{# ==========================================================================
+
+   text_introduction.render()
+
+   ==========================================================================
+
+   Description:
+
+   Create a Text Introduction molecule.
+   See [GHE]/flapjack/Modules-V1/wiki/Text-Introduction
+
+   settings:
+
+      heading:   Heading text.
```

Comment:
Should include updates in https://github.com/cfpb/cfgov-refresh/pull/1062/files#diff-6b654a4934f0c0090d18d76a7c1e8b88R14